.github/workflows/release.yml

```diff
@@ -39,15 +39,19 @@ jobs:
           node-version: '14.x'
           registry-url: 'https://registry.npmjs.org'
 
+      # We need llvm-lipo utility from LLVM to build a universal mac binary.
+      - run: wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh && sudo ./llvm.sh 16
       - run: ./scripts/install_zig.sh
-      - run: ./zig/zig build dist -- --build --publish --version 0.9.${{ github.run_number }} --sha ${{ github.sha }}
+      - run: ./zig/zig build release -- --build --publish --version=0.14.${{ github.run_number }} --sha=${{ github.sha }}
         env:
           NUGET_KEY: ${{ secrets.NUGET_KEY }}
-          TIGERBEETLE_GO_GITHUB_TOKEN: ${{ secrets.TIGERBEETLE_GO_GITHUB_TOKEN }}
+          TIGERBEETLE_GO_PAT: ${{ secrets.TIGERBEETLE_GO_PAT }}
```

Comment:
What is "PAT" short for?

Reply:
Personal Access Token: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens

We use this token to clone & push to tigerbeetle-go repostiory over here:

https://github.com/tigerbeetle/tigerbeetle/pull/1236/files#diff-4dfde4c7ef7e6790beb4b0ef50fd5644d15d3dee53eeb2020046eb0de80e11ddL463

Reply:
(the token itself starts with something like `github-pat`, so I consider `PAT` to be an appropriate abbreviation here)